redcap_bridge/test_redcap/test_utils.py

```diff
@@ -0,0 +1,35 @@
+import os
+import pathlib
+import shutil
+import tempfile
+
+import pytest
+
+test_directory = pathlib.Path(tempfile.gettempdir()) / 'diglabtools_testfiles'
+
+
+@pytest.fixture
+def initialize_test_dir(clean=True):
+    """
+    Create main test folder if required and add test files
+
+    Parameters
+    ----------
+    clean: (bool)
+        Remove test folder first in case it exists.
+
+    Returns
+    -------
+    test_directory: (str)
+        path of the test directory
```

Comment:
```suggestion
   str
       path of the test directory
```

Reply:
No, the standard should be 1 tab = 4 spaces, what were you comparing to? maybe the other code would need to have the indentation fixed..

Reply:
Yes sorry it's bit hard in the web interface =/ I've updated the code